### ISSUE TITLE
Rahukalam only return data related to day time.

### ISSUE THREAD

**Issue by shyaminayesh** [sffjunkie/astral#32]
When I use `Location.rahukaalam` It's returning tuple of 2 values for the day time. Is there a way to get night time rahukaalam using this API ?

**Comment by sffjunkie** [comment]
No, not currently. I thought it was only a day time thing, however I see from https://www.drikpanchang.com/panchang/rahu-kaal.html it can also be a night time thing. I am currently refactoring the code so I may add it later.

**Comment by shyaminayesh** [comment]
Hi @sffjunkie, any updates on this ? :smiley:

**Comment by shyaminayesh** [comment]
Hi @sffjunkie ? any updates ?

**Comment by sffjunkie** [comment]
I have added times during the night into version 2.0
